### PR TITLE
Added option to send presentations as attachments to Opencast

### LINKED ISSUE #12: Add presentation slides as attachment
If we have the presentation slides as pdf (and maybe they differ from the default slides), it would be nice to have them as an attachment in Opencast.

### PR DESCRIPTION
**Pull request by Arnei**
Resolves #12

### CODE REVIEW

File: post-archive/post_archive.rb [hunk 1]
```diff
@@ -752,6 +756,23 @@ def cleanup(tmp_path, meeting_id)
...
+      mediapackage = OcUtil::requestIngestAPI($oc_server, $oc_user, $oc_password,
+                    :post, '/ingest/addAttachment', DEFAULT_REQUEST_TIMEOUT,
+                    {:mediaPackage => mediapackage,
+                    :flavor => "presentation/pdf",
+                    :body => File.open(presentationFilePath, 'rb') })
+    end
+  end
+  BigBlueButton.logger.info( "Mediapackage: \n" + mediapackage)
+else
+  BigBlueButton.logger.info( "Adding Presentations as PDFs is either disabled or there were no presentations, skipping adding Presentations as PDFs.")
```
Comment (Rillke) [inline post-archive/post_archive.rb:774]: > or there were no presentations

There is no check whether PDFs exist as far as I can see.

**Review by Rillke — CHANGES_REQUESTED** [review]

File: post-archive/post_archive.rb [hunk 2]
```diff
@@ -54,6 +54,10 @@
 # Suggested default: false
 $sendChatAsSubtitleAttachment = false
 
+# Adds the presentations from a meeting to the attachments in Opencast as PDF files
```
Comment (Rillke) [inline post-archive/post_archive.rb:57]:
```suggestion
# Add all uploaded presentations from a meeting to the attachments in Opencast as PDF files.
# Please make sure to use this feature respectfully.
```

**Review by Rillke — CHANGES_REQUESTED** [review]
Presenters might be surprised to find all their uploads in Opencast.

File: post-archive/post_archive.rb (hunk 1 above)
Comment (Arnei) [reply post-archive/post_archive.rb]: Right, thanks. Changed the log message.

**Pull request merged**